### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by juliasourceress**

### CODE REVIEW

**Review by sqs — APPROVED** [review]

**Review by dadlerj — APPROVED** [review]
Thanks for updating. But no need for the special description and honorifics there 😄 . I'm fine with or without, or with something like "(temporary)" on my name instead. Totally up to you.

File: handbook/ce/index.md [hunk 1]
```diff
@@ -20,9 +20,10 @@ TODO
 
 ## Members
 
-- [Dan Adler](../../company/team/index.md#dan-adler-he-him) (CE team manager)
-- [Christine Lovett](../../company/team/index.md#christine-lovett-she-her)
-- [Tion Thomas](../../company/team/index.md#tion-thomas-he-him)
+- [Julia Gilinets](../../company/team/index.md#julia-gilinets-sheher) (VP of CE)
+- [Dan Adler](../../company/team/index.md#dan-adler-hehim) (Sourcegraph's first ever CE and previous CE team manager)
```
Comment (dadlerj) [inline handbook/ce/index.md:24]:
```suggestion
- [Dan Adler](../../company/team/index.md#dan-adler-he-him) (interim CE member, former CE manager)
```

**Comment by dadlerj** [comment]
Just noticed this wasn't merged. Fixed merge conflicts and updated my title. LMK if you're okay with this @juliasourceress !

**Comment by juliasourceress** [comment]
👍🏻
Thank you!

**Pull request merged**